### PR TITLE
Assignments/number functions

### PR DESCRIPTION
**Pull request by lashlockbhs**

### CODE REVIEW

File: assignments/number-functions/code.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  return (x + y)/2
+}
+function averageOfThree(x, y, z){
+  return (x + y + z)/3
+}
+function distance(x, y){
+  return Math.abs(x-y)
+}
+function manhattanDistance(x1, y1, x2, y2){
+  return Math.abs(x1 - x2) + Math.abs(y1 - y2)
```
Comment (gigamonkey) [inline assignments/number-functions/code.js:26]: To be even more stylish you could have written this to use the `distance` function you just defined.

```suggestion
  return distance(x1, x2) + distance(y1, y2);
```

Makes it slightly easier to read, say for someone who doesn't necessarily know what Manhattan Distance is.

File: assignments/number-functions/code.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+function add(x, y){
```
Comment (gigamonkey) [inline assignments/number-functions/code.js:1]: Please convert all these to the function definition syntax I'm teaching you. As I said in class, this syntax is fine though it's somewhat out of fashion for various subtle reasons, some of which I care about.

File: assignments/number-functions/code.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  return x+y
```
Comment (gigamonkey) [inline assignments/number-functions/code.js:2]: Statements like this should end with a `;`. I'll talk more about this later but Javascript is kinda loose with when it really requires `;`s but it's good practice to put them in.

**Review by gigamonkey — CHANGES_REQUESTED** [review]
Please update the function definition syntax and then tag me by mentioning my handle `@gigamonkey` in a comment so I can take another look.

File: assignments/number-functions/code.js (hunk 3 above)
Comment (lashlockbhs) [reply assignments/number-functions/code.js:2]: Will do

**Pull request merged**